MAINTAINERS.md

```diff
@@ -0,0 +1,282 @@
+- `semver-minor` for enhancements ("features")
+- `semver-major` for backwards-incompatible ("breaking") changes to the *output*, *API*, or environment support
+
+Pull requests don't need `semver-*` labels, *unless* there is no associated issue.
+
+An issue or PR which will introduce a breaking change will be `semver-major`, *regardless* of any other label.
+
+### Questions
+
+Support questions should be answered if possible, but the user should be directed to the chat room, StackOverflow, or Google Groups for further help, *if* it is indeed a Mocha-related issue.  
+
+If it's *not* a Mocha problem (people tend not to believe this), you may want to show a counter-example.  It's often helpful to direct the issue author to the responsible project, if you can determine what that is.
+
+- `common-mistake`: Issues which are *repeatedly* asked support questions or misunderstandings.  This may also apply to questions which receive a lot of 👍's
+- `question`: Add this label if it's just a support question
+
+### Bugs
+
+- `unconfirmed`: A maintainer has not yet or cannot reproduce; typically `needs-feedback` follows (see "Feedback & Follow-ups" below)
+- `bug`: A confirmed bug
+
+### Subsystems, Environments, Etc.
+
+- `reporter`: Usually concerning Mocha's output
+- `browser`: Issues unique to a browser environment
+- `documentation`: Issues around incorrect or missing docs
+- `qa`: Issues around Mocha's own test suite
+- `chore`: Refactors, CI tweaks, dependencies, etc.
+- `developer experience`: Issues which will make it easier to contribute and maintain Mocha
+
+### Feedback & Follow-ups
+
+Issues or PRs which require action or feedback from a *specific* maintainer, should have that item *assigned* to them.
+
+- `needs-feedback`: Issues which *need feedback from the original author*.  Will automatically become `stale` (see "Meta" section below)
+- `needs-review`: For *pull requests only*; PRs which need a maintainer to inspect and/or merge them
+
+### Meta
+
+- `stale`: The "stalebot" marks things as stale and will close issues if they need feedback but haven't received any.  Comment on an issue to prevent this from happening.
+- `duplicate`: Any issue which is a duplicate of some other issue.  Label `duplicate`, write "duplicate of #<ISSUE>" in a new comment, and close.  [Read more about marking issues as duplicates](https://help.github.com/articles/about-duplicate-issues-and-pull-requests/)
+
+## Milestones
+
+A major release following SemVer is "just a number".  Yet, given the vast amount of projects which consume Mocha, we should avoid *frequent* breaking changes, as this becomes disruptive.
+In the manner of "ripping off a band-aid", sometimes we will want to group features or breaking changes together.
+If so, we can add those issues and/or PRs to a new GitHub "milestone", to keep track of what needs to go in before we release.
+Typically milestones have not been useful for anything other than grouping breaking changes together.
```

Comment:
Is the formatting correct here?
There are line breaks, but niether bullet points nor empty lines like paragraphs.
That may render neatly and be intentional, just double-checking.

Reply:
will double-check myself